src/include/network/lan/bridge.rb

```diff
@@ -60,6 +87,21 @@ def configure_as_bridge_port(device)
 
       NetworkInterfaces.Commit
       NetworkInterfaces.Add
+
+      NetworkInterfaces.Current = selected_interface
+    end
+
+    # Asks the user about adapt the current bridge port config in case that it
+    # is configured in the old way (BOOTPROTO == static) & (IPADDR == 0.0.0.0).
+    def fix_bridge_port_config?(ports)
```

Comment:
Maybe `adapt_bridge_port_config?` could be a better name for this method, and then instead of configure_as_bridge_port define this method as it is an adaption and not a new configuration at all:

```
def adapt_bridge_port_config?(ports)
  ...
end
 
def adapt_bridge_port_config!(ifcfg_name)
  item_id = LanItems.get_configured(ifcfg_name)
  devmap = LanItems.GetDeviceMap(item_id)

  return false unless devmap

  devmap["IPADDR"] = ""
  devmap["NETMASK"] = ""
  devmap["BOOTPROTO"] = "none"
  # take out PREFIXLEN from old configuration (BNC#735109)
  devmap["PREFIXLEN"] = ""

  LanItems.SetDeviceMap(item_id, devmap)

  true
end
```

@mchf what do you think?